ui/apps/notifications/src/components/newComponents/NotificationsPage.tsx

```diff
@@ -0,0 +1,305 @@
+import React from 'react';
+import { tw } from '@twind/core';
+import { useTranslation } from 'react-i18next';
+import { useSearchParams } from 'react-router-dom';
+import { useGetLogin, useFetchNotifications, useMarkAsRead } from '@akashaorg/ui-awf-hooks';
+import Button from '@akashaorg/design-system-core/lib/components/Button';
+import Card from '@akashaorg/design-system-core/lib/components/Card';
+import List, { ListProps } from '@akashaorg/design-system-core/lib/components/List';
+import Icon from '@akashaorg/design-system-core/lib/components/Icon';
+import NotificationsCard from '@akashaorg/design-system-core/lib/components/NotificationsCard';
+import Snackbar, { SnackBarType } from '@akashaorg/design-system-core/lib/components/Snackbar';
+import Stack from '@akashaorg/design-system-core/lib/components/Stack';
+import Tab from '@akashaorg/design-system-core/lib/components/Tab';
+import Text from '@akashaorg/design-system-core/lib/components/Text';
+import { EntityTypes, RootComponentProps } from '@akashaorg/typings/ui';
+import routes, { SETTINGS_PAGE, CUSTOMIZE_NOTIFICATION_WELCOME_PAGE } from '../../routes';
+
+interface Notification {
+  id: string;
+  [key: string]: any;
+}
+
+const NotificationsPage: React.FC<RootComponentProps> = props => {
+  const [searchParams] = useSearchParams();
+  const [message, setMessage] = React.useState('');
+  const [messageType, setMessageType] = React.useState('');
+  const [showMenu, setShowMenu] = React.useState(false);
+
+  React.useEffect(() => {
+    /* uncomment this part when backend data becomes available */
```

Comment:
for areas that we need to get back later to modify please leave a comment with @TODO so we can easily find them